### PR TITLE
Fix permission error in Elastic Beanstalk when writing to config.json…

### PR DESCRIPTION
**Pull request by fabioespinosa**
#### Summary
It took me a while to figure out that I needed to change the permissions locally of the config.json file for Elastic Beanstalk later to not have any issue reading and writing to the file. The change is preserved when compressing the file.

### CODE REVIEW

File: contrib/aws/README.md [hunk 1]
```diff
@@ -13,3 +13,6 @@ eb init
 eb create prod
 eb open prod
 ```
+
+### Note
+You must chmod the config.json file, then compress the whole folder starting form mattemost-docker/contrib/aws/* and then upload the .zip file to elastic beanstalk.
```
Comment (jfrerich) [inline contrib/aws/README.md:18]: Hope you don't mind the drive-by comment. :)
```suggestion
You must chmod the config.json file, then compress the whole folder starting from mattermost-docker/contrib/aws/* and then upload the .zip file to elastic beanstalk. 
```

**Comment by cpanato** [comment]
the title of this PR is not 100% correct, this is not fixing anything, instead of you are adding some notes in the documentation.

**Comment by jasonblais** [comment]
hey @fabioespinosa, and thanks for your contribution! Let us know if you have any questions about Jelena's feedback above? We'd be happy to help!

**Comment by TheStu** [comment]
I don't have a lot of experience with command line / EB so many I'm missing something obvious, but I'm stuck in the same place and the instructions listed here don't seem to be helping. Specifically I've:

- run `chmod 777 config.json` in `/mattermost-docker/contrib/aws/app/mattermost/config`
- zipped everything in the `\mattermost-docker\contrib\aws` folder together
- uploaded the zip to my EB environment via the 'Upload and deploy' button in the EB dashboard.

But still getting this error in the EB app logs:

`Error: failed to load configuration: unable to load on store creation: failed to persist: failed to write file: open /mattermost/config/config.json: permission denied`

Am I missing something obvious here?

**Comment by lindy65** [comment]
Hi @fabioespinosa - I'm helping to review stale PRs and, since we haven't heard from you in 6 months, I'll close this PR off for now. Please feel free to re-open if you're able to prioritize. Thanks!

//cc @cpanato @jfrerich
